### DISCUSSION TITLE
2023%EB%85%84-%ED%9A%8C%EA%B3%A0-%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8-%ED%87%B4%EC%82%AC-%EC%B0%BD%EC%97%85-%EC%B1%84%EC%9A%A9-%EA%B8%B0%EC%88%A0/

### DISCUSSION THREAD

**Discussion by giscus** [heli-os/profile-giscus#5] [Comments]
# 2023%EB%85%84-%ED%9A%8C%EA%B3%A0-%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8-%ED%87%B4%EC%82%AC-%EC%B0%BD%EC%97%85-%EC%B1%84%EC%9A%A9-%EA%B8%B0%EC%88%A0/

볼타 CTO 테오 블로그

https://dataportal.kr/2023%EB%85%84-%ED%9A%8C%EA%B3%A0-%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8-%ED%87%B4%EC%82%AC-%EC%B0%BD%EC%97%85-%EC%B1%84%EC%9A%A9-%EA%B8%B0%EC%88%A0/

**Comment by lleellee0** [comment]
개추드립니다

**Reply by heli-os** [reply]
감사합니다.

**Comment by SightStudio** [comment]
재밌게 잘읽었습니다.

**Reply by heli-os** [reply]
감사합니다.

**Comment by w-beom** [comment]
중간중간에 고민했던 흔적이나 생각하셨던 원칙 같은 것들이 공감 가는 게 많네요.
재미있게 잘 읽었습니다. 2024년에도 파이팅!

**Reply by heli-os** [reply]
감사합니다. 2024년도 파이팅입니다!

**Comment by mingg123** [comment]
안녕하세요. 잘 읽었습니다.
읽다가 한가지 궁금한 점이 들어서 댓글을 남기게 되었습니다. 엑셀 파싱 같은 경우 유효성 검사를 백-> 프론트로 하셨다고 해주셨는데
백에서 해야할 작업을 프론트에서 했다 정도로 생각이 드는데요. 프론트로 했을때 백에서 했을때보다 어떤 이점이 더 있는걸까요? 
유효성 검사가 완료되기 이전에 불필요한 파일을 서버로 전송하지 않는다 외에 더 추가로 있을까요? 잘 몰라서 여쭈어봅니다!  감사합니다!

**Reply by heli-os** [reply]
`1) 파일 업로드/검증`, `2) 검증된 데이터를 기반으로 후속 처리`까지의 과정이 모두 '하나의 요청'으로 되어 있다면 큰 문제는 없습니다.

다만, 이 둘이 다른 요청으로 이루어지는 경우 프로세스가 비교적 복잡해집니다. 저희는 이 둘이 서로 다른 요청으로 이루어지는 방식이었고, 설계 방식에 따라 다를 수 있지만 크게 두 가지 접근 방식을 고려했었습니다.

1. 파일 업로드 -> 서버 검증 -> 검증된 데이터 서버에 임시 저장 -> 그 이후 FE에서 다른 값과 함께 서버에 작업 요청
  이 경우 서버에서 검증된 데이터를 임시 저장하고, 이를 관리하기 위한 상태를 관리해야만 합니다.
2. 파일 업로드 -> 서버 검증 -> 검증된 데이터 응답 -> FE에서 검증된 데이터 + 다른 값과 함께 서버에 작업 요청
  이 경우 서버에서 상태를 관리 할 필요는 없어졌지만, 검증된 데이터를 주고 받아야합니다. 보통 엑셀로 업로드하게 되면 대용량이기 때문에 과도한 네트워크 i/o가 발생할 수 있습니다.

**Reply by mingg123** [reply]
그런 니즈가 있었군요 
자세히 답변 해주셔서 감사합니다. 😊

**Comment by dd55-na** [comment]
안녕하세요, 한편의 다큐멘터리 같이 너무 재미있게 잘 봤습니다. 24년에는 볼타 더 성장하길 응원합니다.

한 가지 궁금한게 있는데요. 
'노코드 랜딩 페이지를 '이러한 기능이 제공될 예정인데 지금 신청하면 무료, 4주 뒤에는 유료!'라는 뉘앙스로 꾸미기 시작했고, 200곳 가량이 사용 신청을 해주셨습니다. '

200곳 가량 사용 신청을 어떻게 획득하셨는지 궁긍합니다. 아마 ceo분께서 랜딩페이지 홍보를 하셨을거라 추측되는데 어디에 어떻게 홍보하셨는지 자세하게 알려주시면 고맙겠습니다. 저도 예비창업자라 이 부분이 항상 고민이었거든요 :)

**Reply by heli-os** [reply]
노코드 랜딩페이지를 만들었을 때 기준으로 저희는 법인이 없었고, 광고/마케팅 전략을 실행하기 위한 비용도 없었습니다.

그래서 저와 공동창업자가 알고 지내던 주변의 창업가, CEO, CFO, 재무담당자, 사업담당자 등에게 1:1로 연락을 돌리기도 하고 관련 종사자들이 모여있는 커뮤니티(오픈채팅방 등)에 포스팅하기도 했습니다. 내용은 설문이나 수요 조사 같은게 아니라 실제 서비스 신청을 해달라는 내용이었구요. 내용을 구성할 때에는 무조건 프로페셔널하게 보여야한다고 생각했습니다.

다들 많이 이용하시는 방법이지만 그나마 성과를 낼 수 있었던 이유로는 `타겟 세그멘테이션을 정말 구체적으로 했다는 부분`과 `저희가 직관적으로 세웠던 가설이 꽤 유의미했기 때문`이 아닐까 싶습니다.

저는 이전에 100개 이상의 사이드 프로젝트를 해보면서 다양한 가설을 세우고, 검증해보는 경험과 얼리 스테이지 스타트업 합류를 목적으로 200곳 이상의 스타트업과 비즈니스에 대한 이야기를 나눠본 경험이 있었습니다.

이런 경험이 쌓이면서 본능적으로 행동했던 것 같네요.

**Reply by dd55-na** [reply]
우와 상세한 답변 감사드립니다.

역시 내공이 엄청나시네요. 주변에 볼타 홍보하며 계속 응원하겠습니다. 
화이팅하세요 :D

**Comment by gosekose** [comment]
볼타 화이팅~~~ 테오 글렌 메이 제이 셰인 화이팅~~~

**Reply by heli-os** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Comment by jdm85kor** [comment]
공유 감사합니다. 종종 이야기 들으러 오고 싶습니다^^
